gtsfm/scene_optimizer.py

```diff
@@ -413,13 +422,15 @@ def save_full_frontend_metrics(
         # Note: if GT is unknown, then R_error_deg, U_error_deg, and inlier_ratio_gt_model will be None
         metrics_list.append(
             {
-                "i1": i1,
-                "i2": i2,
+                "i1": int(i1),
```

Comment:
turns out they are int64, not int. So json can't serialize it otherwise. It's strange, and I'm not 100% sure why. But this fixes the serialization error.

Reply:
wasnt this working before this change?

Reply:
they were working before, right? so what changed?